Description: FreeBSD platform integration (patch-hw_xfree86_os-support_meson.build).
Origin: FreeBSD ports, https://raw.githubusercontent.com/freebsd/freebsd-ports/0433da2a516ae48cce1e2d59e97df4ebbef43afe/x11-servers/xorg-server/files/patch-hw_xfree86_os-support_meson.build
Forwarded: not-needed (BSD platform integration)

--- a/hw/xfree86/os-support/meson.build
+++ b/hw/xfree86/os-support/meson.build
@@ -96,6 +96,8 @@ elif host_machine.system().endswith('bsd')
         'bsd/bsd_VTsw.c',
         'bsd/bsd_bell.c',
         'bsd/bsd_init.c',
+        'linux/lnx_platform.c',
+        'misc/SlowBcopy.c',
 	'shared/pm_noop.c'
     ]
 
@@ -111,7 +113,7 @@ elif host_machine.system().endswith('bsd')
         if host_machine.system() == 'netbsd' or host_machine.system() == 'openbsd'
             os_dep += cc.find_library('i386')
         endif
-    elif host_machine.cpu_family() == 'arm'
+    elif host_machine.cpu_family() == 'arm' or host_machine.cpu_family() == 'aarch64'
         srcs_xorg_os_support += 'bsd/arm_video.c'
     elif host_machine.cpu_family() == 'ppc' or host_machine.cpu_family() == 'ppc64'
         srcs_xorg_os_support += 'bsd/ppc_video.c'
